source/_integrations/smtp.markdown

```diff
@@ -98,7 +98,9 @@ notify:
     sender_name: "My Home Assistant"
 ```
 
-Keep in mind that Google has some extra layers of protection which need special attention (Hint: 'Less secure apps'). If you have 2-step verification enabled on your Google account, you'll need to use [an application-specific password](https://support.google.com/mail/answer/185833?hl=en).
+Keep in mind that Google has some extra layers of protection which need special attention. By default, the usage by external applications is limited so you will need to visit the [less secure apps](https://www.google.com/settings/security/lesssecureapps) page and enable it to be able to send e-mails, and be aware that Google will periodically turn it off if it is not used (no e-mail is sent).
+
+To avoid having your e-mail notifications broken due to the less secure apps behaviour, it is recommended that you enable 2-step verification on your Google account, and use [an application-specific password](https://support.google.com/mail/answer/185833?hl=en) in your notification configuration.
```

Comment:
This is very specific content that only applies to a specific email provider. Lets move this down in the documentation in a separate chapter, or even completely remove it.

Reply:
I agree........I just didn't do that as I thought it was up for some reason. I will keep in the top a generic SMTP configuration example and create a chapter at the end of the page for specific e-mail provider configuration examples.